src/parse/converters/expressions/primary/literal/readRegexpLiteral.js

```diff
@@ -0,0 +1,17 @@
+import { REGEXP_LITERAL } from 'config/types';
+
+// as far as I can tell, this only falls over if modifiers are repeated erroneously
+var regexpPattern = /^\/(?:(?=\\\/)..|[^\/])+\/[gimy]{0,4}/;
```

Comment:
AFAIK those are all perfectly legal. I've actually started with refactoring the parser (a while ago, but didn't have time for Ractive lately) and I have a working regExp parser  (at least I think so). I can look for the relevant code and submit a PR if you want.